### PR TITLE
Update Copyright

### PR DESCRIPTION
**Pull request by crunchymaggie**
# Description  

Update copyright year to 2022

Fixes #  

Depends on #  

## Type of change  
Please check all options that are relevant  
- [ ] Bug fix (change which fixes an issue)  
- [ ] Feature (change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update only  

# How Has This Been Tested?  
**Tested Configuration**:  
- [ ] Ansible, Specify version(s):  
- Installation method:  
    - [ ] Binary install from source, version:  
    - [ ] OS package repository, distro, and version:  
    - [ ] Local package server, version:  
    - [ ] Custom-built package, version:  
    - [ ] Other:  
- [ ] PostgreSQL, Specify version(s):  
- [ ] docs tested with hugo version(s):  

Tested with playbook(s):  

# Checklist:  
- [ ] My changes generate no new lint warnings  
- I have made corresponding changes to:  
    - [ ] the documentation  
    - [ ] the release notes  
    - [ ] the upgrade doc

### CODE REVIEW

**Comment by sharmay** [comment]
@crunchymaggie Missing lot of updates and Duplicate of 267 ?\

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2017-2021 Crunchy Data Solutions, Inc.
+Copyright 2017-2022 Crunchy Data Solutions, Inc.
```
Comment (hunleyd) [inline LICENSE:1]: all rights reserved?

**Review by hunleyd — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (keithf4) [reply LICENSE:1]:
```suggestion
Copyright 2017-2022 Crunchy Data Solutions, Inc. All Rights Reserved.
```

**Review by keithf4 — CHANGES_REQUESTED** [review]

**Comment by pgguru** [comment]
Didn't #267 already take care of this?

**Review by keithf4 — APPROVED** [review]

**Comment by keithf4** [comment]
> Didn't #267 already take care of this?

Looks like the other PR missed the one this touches

**Comment by keithf4** [comment]
> > Didn't #267 already take care of this?
> 
> Looks like the other PR missed the one this touches

Nevermind sorry, was looking at main branch not dev. This was already taken care of

**Comment by keithf4** [comment]
Closing this PR since it was taken care of with #267

**Pull request closed without merging**